cpython-windows/build.py

```diff
@@ -1700,10 +1722,18 @@ def build_cpython(
             log("copying %s to %s" % (source, dest))
             shutil.copyfile(source, dest)
 
-        shutil.copyfile(
-            cpython_source_path / "Tools" / "scripts" / "run_tests.py",
-            out_dir / "python" / "build" / "run_tests.py",
-        )
+        # CPython 3.13 removed `run_tests.py`, we provide a compatibility script
+        # for now.
```

Comment:
How is this used? Is it just something we invoke later on?

Reply:
We don't invoke it. It's placed in the distribution archive for consumers and there's a `run_tests` field in the `PYTHON.json` metadata which points to this script. This is briefly mentioned in the pull request summary, but I think we'll probably open an issue to just remove this in the future, allow it to be null in the metadata, or provide a command in the metadata instead of a path? All of these alternatives are technically breaking changes to the distributions though.

Reply:
For tracking, here was the first commit I did to get past this change https://github.com/indygreg/python-build-standalone/pull/264/commits/5afde6d9fd0272cb94724f26bf58f5ee2d42e6a4 — here we resolve the TODO there and provide the compatibility script.

Reply:
I tested the script on Linux, but not Windows.